### ISSUE TITLE
Converting Doc file to images

### ISSUE THREAD

**Issue by sandipiweb** [mozilla/pdf.js#15702]
is there any way, Creating DOC file thumbnails

**Comment by Snuffleupagus** [comment]
No, which should be obvious if you read [the first sentence of the README](https://github.com/mozilla/pdf.js#pdfjs-):
> [PDF.js](https://mozilla.github.io/pdf.js/) is a Portable Document Format (PDF) viewer that is built with HTML5.
